docs/player_shapeshift_model.md

```diff
@@ -0,0 +1,181 @@
+# player_shapeshift_model
+
+[<-Back-to:World](database-world)
+
+**The \`player_shapeshift_model\` table**
+
+This table holds the information on what values are used for the druid shapeshift models, based on the shapeshift, race, character customisation and their gender.
+
+**Table Structure**
+
+| Field                              | Type    | Attributes | Key | Null | Default | Extra | Comment |
```

Comment:
Hyperlinks for this table? If so, may need to be fixed.

Reply:
If you're asking if the hyperlinks work, they dont work in github but they are set correctly if not mistaken.

Reply:
Disregard, I missed the [] in the table was mb